cfpb/cfgov-refresh
# CCDB: update Landing Map page to use date metadata

This pr fixes an issue where the date used in the tile map legend is not accurately reflecting the last indexed complaint date.  
This fix uses the steps taken in CCDB-UI to pull down the information from the metadata file.

## Additions
- Uses the metadata file to determine when complaints were last index
- tests

## How to test this PR

1. pull down, run ./frontend.sh
2. navigate to: http://localhost:8000/data-research/consumer-complaints/
3. open developer console in chrome 
4. type window.complaint_public_metadata
5. you should see an object with the date info


## Screenshots
<img width="768" alt="Screen Shot 2020-08-05 at 12 23 53 PM" src="https://user-images.githubusercontent.com/1038306/89444223-5e0ba180-d71f-11ea-9e65-eb4fd716396f.png">

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
